src/tests/electron/tests/automated-checks-view.test.ts

```diff
@@ -30,13 +36,20 @@ describe('AutomatedChecksView', () => {
         expect(collapsibleContentElements).toHaveLength(0);
     });
 
+    async function countHighlightBoxes(): Promise<number> {
+        const boxes = await automatedChecksView.client.findElements(ScreenshotViewSelectors.highlightBox);
```

Comment:
consider adding a `ViewController.findElements` to wrap the `.client.$` bit to more clearly give back an `ElementController[]` type

Reply:
discussed offline, deferring